src/Transition.js

```diff
@@ -442,9 +445,11 @@ Transition.propTypes = {
    * }}
    * ```
    *
-   * If the value of appear is not set, then the value from enter is taken.
+   * If the value of `appear` is not set, then the value from enter is taken.
+   *
+   * If the `enter` or `exit` value is `null` or `undefined`, then the timer is set to `0`
    *
-   * @type {number | { enter?: number, exit?: number }}
+   * @type {number | { enter: number, exit: number, appear?: number }}
```

Comment:
Make `enter` and `exit` optional again. And thanks for adding `appear` timeout to the docs too! 👏

Reply:
Ok